source/extensions/common/wasm/context.cc

```diff
@@ -1136,7 +1141,13 @@ bool Context::validateConfiguration(absl::string_view configuration,
   return result;
 }
 
-absl::string_view Context::getConfiguration() { return configuration_; }
+absl::string_view Context::getConfiguration() {
+  if (plugin_) {
```

Comment:
This breaks validateConfiguration.

Reply:
How come?

Reply:
Then let's just remove the configuration_ member.

Reply:
Done.